Readme.rst

```diff
@@ -84,8 +84,6 @@ Events                    Add event start, duration, and location info to post m
 
 Extract table of content  Extracts table of contents (ToC) from ``article.content``
 
-Feed Summary              Allows article summaries to be used in ATOM and RSS feeds instead of the entire article
-
```

Comment:
This ReadMe change is from a commit that has already landed in master and should not be present in this pull request.